spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -464,3 +464,28 @@ for (Pageable p = Pageable.ofSize(25).sortBy(Sort.desc("yearBorn"), Sort.asc("na
 }
 ----
 
+== Jakarta Data Providers
+
+=== Modes
+
+Jakarta Data Providers implement the Jakarta Data API and specification requirements for one or more of the following modes.
+
+==== Standalone Mode
+
+Standalone mode means that the Jakarta Data provider is able to run on its own outside of a Jakarta EE platform or profile. Standalone Jakarta Data providers can choose whether or not to provide their own custom dependency injection leveraging `@Inject`, whether or not to provide extensions to tie into dependency injection via Jakarta Contexts and Dependency Injection (CDI) SE, whether or not to tie into the Jakarta Data Provider SPI, and whether or not to define other vendor specific ways of accessing Jakarta Data repositories. The Jakarta Data TCK includes a Standalone mode for validating Jakarta Data Providers that requires at least one of the first three of the aforementioned approaches.
```

Comment:
This section very throughly explores the different options for a Jakarta Data Provider for Standalone implementations.  When it comes to the TCK I think we'll need some more documentation on the order in which we try to discover the provider and what happens if none is found. 

For now, I'll tag issue #119 but will likely open a new issue for this once this is finalized and merged. 
Thank you.